google-datacatalog-rdbms-connector/tests/google/datacatalog_connectors/rdbms/scrape/config_test.py

```diff
@@ -16,8 +16,8 @@
 
 import os
 import unittest
-import mock
 
+import mock
```

Comment:
Please keep this statement in the previous block and keep it alphabetically ordered.